libs/dao/src/main/java/com/akto/dao/ApiInfoDao.java

```diff
@@ -53,6 +66,124 @@ public void createIndicesIfAbsent() {
             ApiInfoDao.instance.getMCollection().createIndex(Indexes.ascending(fieldNames));    
             counter++;
         }
+
+        if (counter == 4) {
+            String[] fieldNames = {ApiInfo.LAST_SEEN};
+            ApiInfoDao.instance.getMCollection().createIndex(Indexes.descending(fieldNames));    
+            counter++;
+        }
+
+        if (counter == 5) {
+            String[] fieldNames = {ApiInfo.LAST_TESTED};
+            ApiInfoDao.instance.getMCollection().createIndex(Indexes.descending(fieldNames));    
+            counter++;
```

Comment:
no need to use counter. Mongo discards any createIndex query if an index is already present. No need for optimization here